src/Analyzer.Core.JsonRuleTests/Tests/AppServiceApiApp_HTTPS.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
It would be nice if we can add some info on what scenario the result would fall under. For example: 
- The line X result is in the case where it's a finding in a dependsOn resource. 
- Line Y is a result from the base case of the evaluation. 
- Line Z is the case where 1 of 3 leaf expressions failed in an allOf operator.

Reply:
Do you mean adding comments like that in the test config?

Reply:
Yea comments would work I think.

Reply:
@JohnathonMohr Can you add comments for each expected result for the tests you've added?